src/services/random.ts

```diff
@@ -1,5 +1,7 @@
+const opacity = 0.5;
```

Comment:
if you moved it away to variables - you can have them like default values for `randomColor` function params